src/main/scala/edu/ie3/powerFactory2psdm/util/RandomSampler.scala

```diff
@@ -6,19 +6,25 @@
 
 package edu.ie3.powerFactory2psdm.util
 
-import edu.ie3.powerFactory2psdm.config.ConversionConfig
-import edu.ie3.powerFactory2psdm.config.ConversionConfig.GenerationMethod
-import org.apache.commons.math3.distribution.NormalDistribution
+import edu.ie3.powerFactory2psdm.generator.ParameterSamplingMethod
+import edu.ie3.powerFactory2psdm.generator.ParameterSamplingMethod.{
+  Fixed,
+  NormalDistribution,
+  UniformDistribution
+}
+import org.apache.commons.math3.distribution.{
+  NormalDistribution => MathNormalDistribution
+}
 
 object RandomSampler {
 
-  def sample(generationMethod: GenerationMethod): Double =
+  def sample(generationMethod: ParameterSamplingMethod): Double =
     generationMethod match {
-      case ConversionConfig.Fixed(value) => value
-      case ConversionConfig.UniformDistribution(lowerBound, upperBound) =>
+      case Fixed(value) => value
+      case UniformDistribution(lowerBound, upperBound) =>
         scala.util.Random.between(lowerBound, upperBound)
```

Comment:
Do you REALLY want this w/o a fixed random seed?

Reply:
Nope, most probably not